test/tests/transfers/general_field/nearest_node/nearest_position/main.i

```diff
@@ -0,0 +1,107 @@
+# Base input for testing transfers. It has the following complexities:
+# - more than one subapp
+# - transfers both from and to the subapps
+# - both nodal and elemental variables
+# Tests derived from this input may add complexities through command line arguments
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[AuxVariables]
+  [to_sub]
+    [InitialCondition]
+      type = FunctionIC
+      function = '1 + 2*x*x + 3*y*y*y'
+    []
+  []
+  [to_sub_elem]
+    order = CONSTANT
+    family = MONOMIAL
+    [InitialCondition]
+      type = FunctionIC
+      function = '2 + 2*x*x + 3*y*y*y'
+    []
+  []
+  [from_sub]
+    initial_condition = -1
+  []
+  [from_sub_elem]
+    order = CONSTANT
+    family = MONOMIAL
+    initial_condition = -1
+  []
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[Problem]
+  solve = false
+  verbose_multiapps = true
+[]
+
+[Outputs]
+  [out]
+    type = Exodus
+    overwrite = true
+    hide = 'to_sub to_sub_elem'
```

Comment:
why hide these?

Reply:
reduce file size! They are set by ICs, nothing interesting happens to them